### PR TITLE
feat: update extension overview

### PR DESCRIPTION
**Pull request by Mause**

### CODE REVIEW

File: docs/extensions/overview.md [hunk 1]
```diff
@@ -26,8 +14,41 @@ INSTALL 'fts';
 LOAD 'fts';
 ```
 
-<!--
-TODO: How to check which extensions have been loaded or are available
+If you are using the Python API client, you can install and load them with the `load_extension(name: str)` and `install_extension(name: str)` methods.
 
-https://github.com/duckdb/duckdb/search?q=loaded_extensions
+## Listing core extensions
```
Comment (Mause) [inline docs/extensions/overview.md:19]: Feedback on these headers is very welcome 😅

Comment (Mytherin) [reply docs/extensions/overview.md:19]: Perhaps just `Listing extensions`? It lists all installed extensions regardless of their source.

File: docs/extensions/overview.md [hunk 2]
```diff
@@ -26,8 +14,41 @@ INSTALL 'fts';
...
+│ postgres_scanner │ false  │ false     │                                                                          │ Adds support for reading from a Postgres database                    │
+│ sqlite_scanner   │ false  │ false     │                                                                          │ Adds support for reading SQLite database files                       │
+│ substrait        │ false  │ false     │                                                                          │ Adds support for the Substrait integration                           │
+│ tpcds            │ false  │ false     │                                                                          │ Adds TPC-DS data generation and query support                        │
+│ tpch             │ true   │ true      │ (BUILT-IN)                                                               │ Adds TPC-H data generation and query support                         │
+└──────────────────┴────────┴───────────┴──────────────────────────────────────────────────────────────────────────┴──────────────────────────────────────────────────────────────────────┘
+```
+
+## All available extensions
```
Comment (Mytherin) [inline docs/extensions/overview.md:40]: Perhaps replace this section with a table auto-generated from the `duckdb_extensions` function, e.g. (in the shell):

```sql
.mode markdown
select extension_name, description from duckdb_extensions();
```


|  extension_name  |                             description                              |
|------------------|----------------------------------------------------------------------|
| fts              | Adds support for Full-Text Search Indexes                            |
| httpfs           | Adds support for reading and writing files over a HTTP(S) connection |
| icu              | Adds support for time zones and collations using the ICU library     |
| json             | Adds support for JSON operations                                     |
| parquet          | Adds support for reading and writing parquet files                   |
| postgres_scanner | Adds support for reading from a Postgres database                    |
| sqlite_scanner   | Adds support for reading SQLite database files                       |
| substrait        | Adds support for the Substrait integration                           |
| tpcds            | Adds TPC-DS data generation and query support                        |
| tpch             | Adds TPC-H data generation and query support                         |

**Review by Mytherin — COMMENTED** [review]
LGTM, minor comments:

File: docs/extensions/overview.md (hunk 2 above)
Comment (Mause) [reply docs/extensions/overview.md:40]: Sure, only issue with that is the table-of-contents are seems to be auto appended by something

**Comment by Mause** [comment]
@Alex-Monahan any chance you could glance over this this week?

**Comment by Alex-Monahan** [comment]
I didn't run it through jekyll, but the content looks great! Thanks!

**Comment by Mytherin** [comment]
Thanks, looks good!

**Pull request merged**